package.json

```diff
@@ -76,6 +76,7 @@
           "@typescript-eslint/consistent-type-imports": [
             "error"
           ],
+          "no-shadow": "warn",
```

Comment:
If we're going to have a rule that we want people to follow, then I would recommend making this an error so that it gets enforced on new code.